app/assets/stylesheets/components/_session_form.scss

```diff
@@ -0,0 +1,39 @@
+.session_form{
+  align-items: center;
+  display: flex;
+  font-weight: 600;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  &__card{
+    background-color: white;
+    border: 1px solid #00000020;
+    border-radius: 3%;
+    margin: 5%;
+    padding: 3%;
+    
+    &__link{
+      color: $text-gray;
+      font-size: 15px;
+      text-decoration: none;
+      
+      &:hover{
+        color: $active-red;
+      }
+    }
+
+    @media (orientation:landscape){
+      width: 30%;
+    }
+  }
+
+  .field{
+    margin: 3% 0% 3% 0%;
```

Comment:
you can simplify this as follows: margin: 3% 0;
Here you can find more information: https://developer.mozilla.org/es/docs/Web/CSS/margin